src/css/index.scss

```diff
@@ -1,3 +1,92 @@
+//@use "sass:map";
+
+// 1. Include functions first (so you can manipulate colors, SVGs, calc, etc)
+@import "../../node_modules/bootstrap/scss/functions";
```

Comment:
I think there must be a way to do these imports such that parcel can bundle in bootstrap without the absolute reference to node modules here.